### PR TITLE
Added radio, checkbox ,slider, images and modal.

### PR DESCRIPTION
**Pull request by j24m**
Added radio, checkbox ,slider, images and modal.

Image component
image

Input component
image

Login modal
image

Signup modal
image

### CODE REVIEW

File: Components/image.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.img-container{
+    width:200px;
```
Comment (mehulsatardekar) [inline Components/image.css]: use rem for sizing if it is more than 10px.

File: Components/image.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    height:200px;
```
Comment (zoef-yt) [inline Components/image.css]: Use rem.

File: Components/input.html [hunk 3]
```diff
@@ -83,52 +86,53 @@ <h1 class="documentation-component-main-heading">INPUT</h1>
...
                     </div>
-                    
+
                     <div class="input-label">
                         <label for="number">
                             Number
                         </label>
-                        
+
                         <input type="number" name="number" placeholder="Enter quantity" />
```
Comment (rahulyadav139) [inline Components/input.html]: you have provided "for" attribute to the label tag but it seems you forgot to add "id" attribute to the subsequent input tag. If you do that then input and label both are connected to each other and you get to focus on input after clicking on the related label tag.

File: Components/input.html [hunk 4]
```diff
@@ -141,6 +145,23 @@ <h1 class="documentation-component-main-heading">INPUT</h1>
...
+                    <input type="radio" name="radio" /><label for="radio">Radio input field</label>
+                </div>
+
+                <div class="checkbox-input">
+                    <input type="checkbox" name="check" /><label for="check">Check input field</label>
+                </div>
+            </div>
+            <div class="component-examples">
+                <div class="range-input">
+                    <label for="volume">Volume<input type="range" min="0" max="100" value="40" name="volume" step="1"
```
Comment (zoef-yt) [inline Components/input.html]: change "for" from volume to slider.

File: Components/modal.css [hunk 5]
```diff
@@ -0,0 +1,120 @@
+.modal-container {
+    width: 100%;
+    height: 100vh;
+    background-color: rgb(187, 187, 245);
```
Comment (zoef-yt) [inline Components/modal.css]: Use var for colour.

File: Components/modal.css [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    color: var(--light-color);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.modal-card {
+    width: 35rem;
+    height: 50rem;
+    box-shadow: 0 0 40px 20px rgba(0, 0, 0, 0.26);
```
Comment (zoef-yt) [inline Components/modal.css]: Use var for colour.

File: Components/modal.css [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+.card-front,
+.card-back {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-position: center;
+    background-size: cover;
+    background-image: linear-gradient(rgba(0, 0, 100, 0.8), rgba(0, 0, 100, 0.8)), url(../Images/Imgcmp.jpg);
```
Comment (zoef-yt) [inline Components/modal.css]: Use var for colour.

File: Components/modal.css [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+    cursor: pointer;
+}
+
+.submit-btn {
+    position: relative;
+}
+
+.submit-btn::after {
+    content: '\27a4';
+    color: #333;
```
Comment (zoef-yt) [inline Components/modal.css]: Use var for colour.

File: Components/modal.html [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+                    <a class="link" href="#">
+                        <li>Text-Utility</li>
+                    </a>
+                    <a class="link link-active" href="#">
+                        <li>Modal</li>
+                    </a>
+                </ul>
+        </div>
+        <div class="documentation">
+            <!-- <h1 class="documentation-component-main-heading">Modal</h1> -->
```
Comment (zoef-yt) [inline Components/modal.html]: Remove commented code.

File: Components/modal.html [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+                                    <input type="checkbox"><span>Remember me</span>
+                                </div>
+                            </form>
+                            <button type="button" class="btn-new-here" onclick="openLogin()">I've an Account</button>
+
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!-- <div class="code-snippet"></div> -->
```
Comment (zoef-yt) [inline Components/modal.html]: Remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: Components/modal.css [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+    border: 1px solid var(--light-color);
+    margin: 6px 0;
+    height: 3.2rem;
+    border-radius: 1rem;
+    padding: 0 1rem;
+    box-sizing: border-box;
+    outline: none;
+    text-align: center;
+    color: var(--light-color);
+}
```
Comment (rahulyadav139) [inline Components/modal.css]: That card with 3D effect is dope. It looks awesome. Great work.

**Review by rahulyadav139 — APPROVED** [review]

File: Components/image.css [hunk 12]
```diff
@@ -0,0 +1,23 @@
+.img-container{
+    width:20rem;
+    height:20rem;
+    border:solid var(--secondary-color);
+
+}
```
Comment (iRohitGaur) [inline Components/image.css:6]: extra spacing

File: Components/modal.css (hunk 5 above)
Comment (iRohitGaur) [reply Components/modal.css]: +1. Also use hex everywhere for consistency.

File: Components/modal.html [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+            <h3 class="sidebar-cmp-heading">Quick Installation</h3>
+            <ul class="ul">
+                <a class="link" href="./getstarted.html">
+                    <li>Get Started</li>
+                </a>
+                <h3 class="sidebar-cmp-heading">Components</h3>
+                <ul class="ul">
+                    <a class="link" href="./avatar.html">
+                        <li>Avatar</li>
+                    </a>
```
Comment (iRohitGaur) [inline Components/modal.html:44]: it should be ul --> li --> a

File: Components/modal.html [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+        var modalCard = document.getElementById("modal-card");
+
+        function openRegister() {
+            modalCard.style.transform = "rotateY(-180deg)";
+        }
+
+        function openLogin() {
+            modalCard.style.transform = "rotateY(0deg)";
+        }
+    </script>
```
Comment (iRohitGaur) [inline Components/modal.html:131]: JS code should not be in HTML

**Review by iRohitGaur — APPROVED** [review]
Good to merge with changes 👍🏻

File: Components/input.html (hunk 4 above)
Comment (j24m) [reply Components/input.html]: done.

File: Components/modal.html (hunk 13 above)
Comment (j24m) [reply Components/modal.html:44]: Will keep this in mind and make the changes later as this particular change will have to be done in all the files. Thanks for mentioning this, nobody noticed. :)

File: Components/modal.html (hunk 14 above)
Comment (j24m) [reply Components/modal.html:131]: Yes, I had to do it fast so took this shortcut but will change it later.

**Review by zoef-yt — APPROVED** [review]
The code looks good. The login and sign modal looks absolutely amazing good work on the UI. Good work changing the code according to Rukhikesh suggestions. Good to merge

**Review by mehulsatardekar — APPROVED** [review]

**Pull request merged**